sw/airborne/subsystems/ins/vf_extended_float.c

```diff
@@ -351,3 +351,55 @@ void vff_realign(float z_meas)
   //vff.offset = 0.;
   vff_init(z_meas, 0., 0., 0.);
 }
+
+/*
+  H = [0 1 0];
+  R = 0.1;
+  // state residual
+  yd = vz - H * Xm;
+  // covariance residual
+  S = H*Pm*H' + R;
+  // kalman gain
+  K = Pm*H'*inv(S);
+  // update state
+  Xp = Xm + K*yd;
+  // update covariance
+  Pp = Pm - K*H*Pm;
+*/
+static inline void update_vz_conf(float vz, float conf)
+{
+  const float yd = vz - vff.zdot;
+  const float S = vff.P[1][1] + conf;
+  const float K1 = vff.P[0][1] * 1 / S;
+  const float K2 = vff.P[1][1] * 1 / S;
+  const float K3 = vff.P[2][1] * 1 / S;
```

Comment:
This is not correct, vf_extended has one state more that has to be taken into account here...